plugins/bookshortcuts.koplugin/main.lua

```diff
@@ -0,0 +1,111 @@
+local ConfirmBox = require("ui/widget/confirmbox")
+local DataStorage = require("datastorage")
+local Dispatcher = require("dispatcher")
+local FFIUtil = require("ffi/util")
+local LuaSettings = require("luasettings")
+local PathChooser = require("ui/widget/pathchooser")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local util = require("util")
+local _ = require("gettext")
+
+local BookShortcuts = WidgetContainer:new{
+    name = "bookshortcuts",
+    shortcuts = LuaSettings:open(DataStorage:getSettingsDir() .. "/bookshortcuts.lua"),
+    updated = false,
+}
+
+function BookShortcuts:onDispatcherRegisterActions()
+    for k,v in pairs(self.shortcuts.data) do
+        if util.fileExists(k) then
+            local directory, filename = util.splitFilePathName(k) -- luacheck: no unused
+            Dispatcher:registerAction(k, {category="none", event="BookShortcut", title=filename, filemanager=true, arg=k,})
```

Comment:
Would gestures work only in File browser ? Can't switch to these books from another book ? Could be handy to jump to some reference book.

Reply:
Leftover from my previous implementation (File Browser had a short list).
What would be a better place to put it? Device?

Reply:
Dunno. Probably, we already have "Open previous document" in there. But `Device>` sounds a bit limited, it mostly mean "Anywhere" :)

Reply:
I switched it to General.